src/Uno.UI/Generated/3.0.0.0/Microsoft.UI.Xaml.Automation.Peers/AnimatedVisualPlayerAutomationPeer.cs

```diff
@@ -0,0 +1,19 @@
+#pragma warning disable 108 // new keyword hiding
+#pragma warning disable 114 // new keyword hiding
+namespace Microsoft.UI.Xaml.Automation.Peers
+{
+	#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
+	[global::Uno.NotImplemented]
+	#endif
+	public  partial class AnimatedVisualPlayerAutomationPeer : global::Windows.UI.Xaml.Automation.Peers.FrameworkElementAutomationPeer
```

Comment:
Wanted to confirm if this should be in the UWP head?

Reply:
Yes,these are the WinUI 2 types, so they should be available in Uno too (even if not in implemented yet). That being said, some are now implemented incorrectly in WUX namespace instead of MUX (like PersonPicture). These will need to be moved in breaking changes branch, I added the list in the rollup issue